packages/affiliate/src/Affiliate.php

```diff
@@ -68,11 +76,13 @@ public function get_affiliate_code() {
 	 * @return string The passed URL with the code added.
 	 */
 	public function add_code_as_query_arg( $url ) {
-		if ( '' !== ( $aff = $this->get_affiliate_code() ) ) {
+		$aff = $this->get_affiliate_code();
+		if ( '' !== $aff ) {
 			$url = add_query_arg( 'aff', $aff, $url );
 		}
 		return $url;
 	}
 }
 
-add_action( 'init', array( 'Jetpack_Affiliate', 'init' ) );
+add_action( 'init', array( 'Automattic\Jetpack\Partners\Affiliate', 'init' ) );
```

Comment:
Ideally, this file should only contain the class definition, and no other definitions or executable code. This action should be hooked somewhere else - ideally, where the library is consumed, so it's more obvious.

Reply:
Makes sense. Will initialize class in file where this is used.